src/main/java/codeu/model/data/Message.java

```diff
@@ -67,4 +68,21 @@ public String getContent() {
   public Instant getCreationTime() {
     return creation;
   }
-}
+
+
+  /** Time display */ 
+  public String getTime() {
+    LocalDateTime localDate = LocalDateTime.ofInstant(creation, ZoneId.systemDefault());
+    int hr = localDate.getHour();
+    String AMPM = "";
```

Comment:
Instead of having a String, you could use a boolean here.  

`
if (hr > 12) {
  hr = hr % 12;
  AM = false;
} else {
  AM = true;
}

String Date = ....... + AM ? "AM" : "PM"
`

Reply:
Boolean AM;
    Boolean PM;
    if (hr > 12) {
      hr = hr % 12;
      AM = false;
      PM = true;
    }
    else {
      AM = true;
      PM = false;
    }
    String Date = localDate.getMonth().toString() + " " + localDate.getDayOfMonth() + ", " + localDate.getYear() + " ~ " + hr + ":" + localDate.getMinute() + " " + AM ? "AM":"PM";
	
like this? I'm not sure how AM ? "AM":"PM"; returns AM or PM.